.github/workflows/cairo-zero-ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
         with:
-          version: nightly
+          version: nightly-a79dfaed6fc6f88cda5f314a25d1b484d9d8c051
```

Comment:
This is only a temporary fix 
Nightly are erased when new ones are pushed

Reply:
we'll update back when that will be fixed